modules/_labs/badge/css/lib/badge.scss

```diff
@@ -0,0 +1,53 @@
+@keyframes wdc-badge-grow {
+  from {
+    transform: scale(0.85);
+  }
+
+  to {
+    transform: scale(1);
+  }
+}
+
+@mixin wdc-badge {
+  // base styles
+  align-items: center;
+  animation: wdc-badge-grow 0.2s ease;
+  border-radius: 999px;
+  box-sizing: border-box;
+  display: inline-flex;
+  font-family: $wdc-font-family;
+  font-size: 11px;
+  font-weight: 700;
+  height: 20px;
+  justify-content: center;
+  min-width: 20px;
+  padding: 0 6.5px;
```

Comment:
Another design question: 6.5 seems weird to me

Reply:
Yep. Rick and I talked about that. 

> "6.5px (obviously obscure) that value is just based on Roboto font sizing and default letter spacing."